### PR TITLE
refactor code

### PR DESCRIPTION
**Pull request by hninpwintaung-hpa**
To reduce the overhead of  creating a new database connection each time, I've refactored the code responsible for establishing database connections by placing it within a static block,. This modification leading to improved overall performance.

Could you please review the code change I made and  If everything looks good, Can I merge it into the main branch.Thank you!

### CODE REVIEW

File: src/main/java/com/employee/dao/EmployeeDAOImpl.java [hunk 1]
```diff
@@ -12,18 +12,26 @@
 
 public class EmployeeDAOImpl implements EmployeeDAO {
 
+	public static Connection conn;
```
Comment (YeWin) [inline src/main/java/com/employee/dao/EmployeeDAOImpl.java]: Line no.15 to 21, they should be private.

**Review by YeWin — CHANGES_REQUESTED** [review]
I still can't see unused date import remove. @hninpwintaung-hpa

File: src/main/java/com/employee/dao/EmployeeDAOImpl.java (hunk 1 above)
Comment (hninpwintaung-hpa) [reply src/main/java/com/employee/dao/EmployeeDAOImpl.java]: I have implemented the necessary changes based on your valuable suggestions. Would you kindly consider reviewing the updated work? Your feedback is greatly appreciated. Thank you!

Comment (hninpwintaung-hpa) [reply src/main/java/com/employee/dao/EmployeeDAOImpl.java]: @YeWin 
 I have implemented the necessary changes based on your suggestions. Would you kindly consider reviewing the updated work? Your feedback is greatly appreciated. Thank you!

**Review by YeWin — APPROVED** [review]
Look good to me. @hninpwintaung-hpa

**Pull request merged**